### DISCUSSION TITLE
remove version from compose files

### DISCUSSION THREAD

**Discussion by fragolinux** [louislam/dockge#510] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

latest docker compose (> 2.25.x) deprecated the initial "version X.Y" line, can you remove it from the generated compose files?
and, can you let us customize if we want old docker-compose.yml or new compose.yaml? thanks

### ✔️ Solution

latest docker compose (> 2.25.x) deprecated the initial "version X.Y" line, can you remove it from the generated compose files?
and, can you let us customize if we want old docker-compose.yml or new compose.yaml? thanks

**Comment by JinbaIttai** [comment]
This is a duplicate of issue #505, fixed by #507
